runtime/md5.c

```diff
@@ -37,6 +37,17 @@ CAMLprim value caml_md5_string(value str, value ofs, value len)
   return res;
 }
 
+CAMLprim value caml_md5_bytes(value b, value ofs, value len)
+{
+  struct MD5Context ctx;
+  value res;
+  caml_MD5Init(&ctx);
+  caml_MD5Update(&ctx, &Byte_u(b, Long_val(ofs)), Long_val(len));
+  res = caml_alloc_string(16);
+  caml_MD5Final(&Byte_u(res, 0), &ctx);
+  return res;
```

Comment:
Same here: could you just call the string version above?

Reply:
done